src/components/Records/Record/GeneralInfo.vue

```diff
@@ -66,29 +75,50 @@
       <!--fairsharingRecord.year_creation-->
       <div class="d-flex">
         <b class="mr-2">Year of Creation:</b>
-        <p>{{ currentRecord['fairsharingRecord'].metadata.year_creation }}</p>
+        <p v-if="currentRecord['fairsharingRecord'].metadata.year_creation">
+          {{ currentRecord['fairsharingRecord'].metadata.year_creation }}
+        </p>
+        <NoneFound
+          v-else
+          :string-field="currentRecord['fairsharingRecord'].metadata.year_creation"
+        />
       </div>
       <!--Registry-->
       <div class="d-flex">
         <b class="mr-2">Registry:</b>
         <p>{{ currentRecord['fairsharingRecord'].registry | capitalize }}</p>
       </div>
       <!--Description-->
-      <div class="d-flex align-center">
+      <div class="d-flex align-center mb-4">
         <b class="mr-2">Description:</b>
-        <p>{{ currentRecord['fairsharingRecord'].description | capitalize }}</p>
+        <p class="mb-0">{{ currentRecord['fairsharingRecord'].description | capitalize }}</p>
       </div>
       <!--HomePage-->
       <div class="d-flex">
         <b class="mr-2 mb-4">Home Page:</b>
         <a
+          v-if="currentRecord['fairsharingRecord'].homepage"
           :href="currentRecord['fairsharingRecord'].homepage"
           target="_blank"
         >{{ currentRecord['fairsharingRecord'].homepage }}</a>
+        <NoneFound
+          v-else
+          :string-field="currentRecord['fairsharingRecord'].homepage"
+        />
       </div>
       <!--Developed Countries -->
       <div class="d-flex flex-wrap">
         <b class="mr-2">Countries involved with this resource:</b>
```

Comment:
Bug with countries but i think it's be issued already:
![image](https://user-images.githubusercontent.com/6538906/92380325-dd451a00-f0bd-11ea-97c7-b5c0330c1f3f.png)
for record 1234.